docs/source/release/v6.9.0/Workbench/Bugfixes/36689.rst

```diff
@@ -0,0 +1 @@
+- Fixed crash when opening the `Rebin` algorithm dialog when a group workspace is selected as the input workspace.
```

Comment:
Could this be a `:ref:` link to the algorithm? Though as a side-note, if you want `monospace` style in `.rst` files you need two backticks around the string.
```suggestion
- Fixed crash when opening the :ref:`algm-Rebin` algorithm dialog when a group workspace is selected as the input workspace.
```